lib/services/cloud-build-service.ts

```diff
@@ -59,14 +51,20 @@ export class CloudBuildService implements ICloudBuildService {
 
 		this.$logger.info(`Finished ${buildInformationString} successfully. Downloading result...`);
 
-		const localBuildResult = await this.downloadBuildResult(buildResult, projectSettings.projectDir, outputFileName);
+		const localBuildResult = await this.downloadBuildResult(buildResult, projectSettings.projectDir);
 
 		this.$logger.info(`The result of ${buildInformationString} successfully downloaded. Log from cloud build is:${EOL}* stderr: ${buildResult.Error}${EOL}* stdout: ${buildResult.Output}${EOL}* outputFilePath: ${localBuildResult}`);
 
+		const buildResultUrl = this.getBuildResultUrl(buildResult);
+
 		return {
 			stderr: buildResult.Error,
 			stdout: buildResult.Output,
-			outputFilePath: localBuildResult
+			outputFilePath: localBuildResult,
```

Comment:
I believe the server returns the FullOutput as well, can we get it here as well?